packages/std/src/query/distribution.rs

```diff
@@ -10,7 +10,20 @@ use super::query_response::QueryResponseType;
 #[serde(rename_all = "snake_case")]
 pub enum DistributionQuery {
     // https://github.com/cosmos/cosmos-sdk/blob/4f6f6c00021f4b5ee486bbb71ae2071a8ceb47c9/x/distribution/types/query.pb.go#L792-L795
-    DelegatorWithdrawAddress { delegator_address: String },
+    DelegatorWithdrawAddress {
+        delegator_address: String,
+    },
+    // https://github.com/cosmos/cosmos-sdk/blob/e3482f2d4142c55f9dc3f47a321b56610a11492c/x/distribution/types/query.pb.go#L525-L532
+    #[cfg(feature = "cosmwasm_1_4")]
+    DelegationRewards {
+        delegator_address: String,
+        validator_address: String,
+    },
+    // https://github.com/cosmos/cosmos-sdk/blob/e3482f2d4142c55f9dc3f47a321b56610a11492c/x/distribution/types/query.pb.go#L614-L619
```

Comment:
The `.pb.go` files are auto-generated Go code from the proto files. Could you link the proto API instead? This is higher level and better to read for API documentation.